Makefile

```diff
@@ -23,7 +23,11 @@ CHECK_EXAMPLE_TIMESTAMPS = $(patsubst examples/%.html,build/timestamps/check-%-t
 
 TASKS_JS := $(shell find tasks -name '*.js')
 
-CLOSURE_LIB = $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())')
+ifeq (CYGWIN,$(findstring CYGWIN,$(OS)))
+  CLOSURE_LIB := $(shell cygpath -u $(shell node -e 'process.stdout.write(require("closure-util").getLibraryPath())'))
```

Comment:
elemoine,
Not sure the quotes are quite right in your
CLOSURE_LIB_NODE_CMD = "node -e 'process.stdout.write(require("closure-util").getLibraryPath())'"

It does not work in Cygwin